test/integration/find_local_council_test.rb

```diff
@@ -8,6 +8,10 @@ class FindLocalCouncilTest < ActionDispatch::IntegrationTest
 
   context "when visiting the start page" do
     setup do
+      example_generator = GovukSchemas::RandomExample.for_schema("placeholder", schema_type: "frontend")
```

Comment:
Find your local council is a [special route](https://github.com/alphagov/frontend/blob/master/lib/tasks/publishing_api.rake#L72-L77), not a placeholder.